app.js

```diff
@@ -39,24 +38,46 @@ const unknownCommandHandler = {
     },
 };
 
+/**
+ * 
+ * @param {Message} message 
+ */
+const isNewUser = async (message) => {
+    const dmMessages = await message.channel.messages.fetch();
+    const botMessages = dmMessages.filter(msg => msg.author.bot);
+
+    if (botMessages) {
+        return false;
+    }
+
+    await message.author.send('👋😃 Hello! I\'m here to help you get to know new Fellows from other Pods.')
+    await message.author.send('If you decide to participate, I\'ll pair you with a new person every week for a short, informal, and fun 1-on-1. It\'s a great way to get to know more people when working remotely!');
+    await message.author.send(`🙌 To enroll yourself into this, enter **${prefix}start**. You can also see other available commands by entering **${prefix}help**`);
+
+    return true;
+}
+
 const commandRegistry = buildCommandRegistry();
 
 client.once('ready', () =>  {
     console.log('The bot is now ready!');
 
-    // Runs this function every minute (for testing purposes)
-    // cron.schedule('1 * * * * *', () => {
+    // Runs this function once every monday.
+    cron.schedule('* * * * * 1', () => {
         scheduler(client);
-    // });
+    });
 });
 
-client.on('message', (message) => {
+client.on('message', async (message) => {
     // Handle public mentions
     if (message.channel.type !== 'dm' && message.mentions.members.has(client.user.id)) {
         message.channel.send('**Coffee Buddy** ☕️ is a bot that pairs you with a new Fellow every week so you can make new lifelong friends while working remotely. Send me a private message to get started! ✨');
         return;
     }
 
+    if (message.channel.type !== 'dm') return;
+
+    if (await isNewUser(message)) return;
```

Comment:
Perhaps we want to check if it's an invalid message before checking if it's a new user ? 
isNewUser might take a decent amount of time

Reply:
Ah, we cannot do that because the invalid message checker also checks if the message has a prefix or not. Which should not be done.